nsds_lab_to_nwb/components/tdt/tdt_reader.py

```diff
@@ -75,32 +72,37 @@ def check_stream(self, stream):
         Returns:
             stream_available (bool): whether stream exisits
         """
-        stream_available = True
+        error_message = f"Device or stream '{stream}' not found. Available streams: ["
+        error_message += ", ".join(self.streams)
+        error_message += "]"
         if stream not in self.streams:
-            error_message = 'Device or stream not found. Available streams: '
-            stream_available = False
-            for stream in self.streams:
-                error_message += stream + ', '
-            warnings.warn(error_message)
-        return stream_available
-
-    def get_data(self, stream):
-        """Get specified stream data
-
-        Args:
-            stream (string): stream name
-
-        Returns:
-            mat, meta (np-array, dict): Returns tuple of data matrix (wide form)
-                                        and metadata dictionary (if no stream returns None)
+            if stream not in stream_alternatives.keys():
+                raise ValueError(error_message)
+            else:
+                stream = stream_alternatives[stream]
+        return stream
+
+    def get_data(self, stream, dev_conf):
+        """Get specified data
+
+        Parameters
+        ----------
+        stream: str
+            Stream name
+        dev_conf: (dict) metadata for the device.
+            nwb_builder.metadata['device'][device_name]. Not used for TDT.
```

Comment:
Because TDT will be our default pipeline moving forward, suggest making `dev_conf` an optional kwarg

Reply:
Aha, I just realized that the requirement is the opposite for HTKReader, and I guess that's why you are treating both input arguments as required? I'm keeping this comment above but wouldn't push strongly - feel free to ignore!

Reply:
Yeah, wasn't sure what to do here. I made a change that falls kind of in the middle. Now both args are kwargs, but they have to be specified at kwargs (can't rely on order).